src/containers/NewProject/NewProject.js

```diff
@@ -1,30 +1,127 @@
 import React from 'react';
-import { Button, Modal, Form, Input } from 'antd';
+import { Button, Modal, Form, Input, Steps, message, Menu, Dropdown, Icon, Card, Table, Divider, Tag } from 'antd';
 import './NewProject.css';
 
+
+const Step = Steps.Step;
+
 const ProjectForm = Form.create({ name: 'form_in_modal' })(
   class extends React.Component {
+
+    renderSteps = (getFieldDecorator, current, next, prev, rooms, addRoom, onCreate, name) => {
+
+      const onClick = function ({ key }) {
+        message.info(`Added a ${key}`);
+        addRoom(key)
+      };
+
+      const menu = (
+        <Menu onClick={onClick}>
+          <Menu.Item key="Kitchen">Add Kitchen</Menu.Item>
+          <Menu.Item key="Bedroom">Add Bedroom</Menu.Item>
+          <Menu.Item key="Bathroom">Add Bathroom</Menu.Item>
+          <Menu.Item key="Living Room">Add Living Room</Menu.Item>
+          <Menu.Item key="Outdoors">Add Outdoors</Menu.Item>
+        </Menu>
+      );
+
```

Comment:
Below, dataSource and columns make up the table at 'checkout'. Data source is inputed into the columns at <Table> farther down.